tests/cam/cam.cpp

```diff
@@ -61,11 +60,9 @@ main(int argc, char *argv[])
 		HoughCircles(fm, circles, HOUGH_GRADIENT, 1, fm.rows/4,
 			hough_p1 + 1, hough_p2, RAD_MIN, RAD_MAX);
 
-		for (i = 0; i < circles.size(); i++) {
-			Vec3i c = circles[i];
+		for (auto c : circles)
```

Comment:
Why not `const`? Why not a reference? You are making copies here instead of using values directly.